client/app/scripts/liveblog-edit/module.js

```diff
@@ -243,6 +243,10 @@ define([
         $scope.$watch(angular.bind(this, function () {return this.blogPreferences;}), function(new_value) {
             vm.isSaved = _.isEqual(vm.blogPreferences, vm.blog.blog_preferences);
         }, true);
+        _.templateSettings.interpolate = /{{([\s\S]+?)}}/g;
+        var compiled = _.template(_.trim(document.getElementById('liveblog-embed-template').innerHTML));
```

Comment:
My purpose was to isolate the query to prevent any issue with other view that can declare another `liveblog-embed-template` somewhere else.
Like you prefer. Then better to use `$document` or `angular.element()` with angular.